src/ros_numpy/registry.py

```diff
@@ -32,6 +32,21 @@ def numpify(msg, *args, **kwargs):
 			raise ValueError("Cannot determine the type of an empty Collection")
 		conv = _to_numpy.get((msg[0].__class__, True))
 
+	if not conv:
+		# on messages coming from a rosbag, __class__ instead of
+		# looking like '_sensor_msgs__Image' looks like tmpldS_vh._sensor_msgs__Image
+		# so we try to check for the _type instead ('sensor_msgs/Image')
+		for class_instance, _ in _to_numpy.keys():
+			if msg._type == class_instance._type:
```

Comment:
Comment here saying that this checks that `package/name` is the same for both would be nice

Reply:
Nevermind, the comment about that above is pretty clear